src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -703,6 +703,37 @@ - (nullable NSData *)computePaseVerifier:(uint32_t)setupPincode iterations:(uint
     return result;
 }
 
+- (nullable NSData *)generateAttestationChallengeForDeviceId:(uint64_t)deviceId
```

Comment:
Oh, and I know @woody-apple asked for this to be called "generate..." but that does not seem right.  The attestation challenge either exists or not; it was generated during session setup.  This is just getting its value if it does exist.  @woody-apple given that, what would you suggest as the name?

Reply:
I appreciate the thorough review @bzbarsky-apple. @woody-apple please let me know your thoughts on naming.

Reply:
I am fine with anything, let’s get this in now, and fix naming in our API review.